app/adapters/plan.js

```diff
@@ -1,3 +1,23 @@
 import V3Adapter from 'travis/adapters/v3';
+import { inject as service } from '@ember/service';
+import { reads } from '@ember/object/computed';
 
-export default V3Adapter.extend();
+export default V3Adapter.extend({
+  accounts: service(),
+  account: reads('accounts.user'),
+
+  buildURL(modelName, id, snapshot, requestType) {
+    let url = this._super(...arguments);
+    let route = '/plans_for/user';
+    if (this.account.type === 'organization') {
+      route = `/plans_for/organization/${this.account.id}`;
+    }
+    url = `${url}${route}`;
+    return url;
```

Comment:
No need to re-assing `url` here, you could just `return url + route`. In that case `url` could be `const`